docs/eventing/02-01-eventing-architecture.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Architecture
+---
+
+Kyma Eventing uses NATS to implement an Event Publisher Proxy and Eventing Controller which work together to process and deliver events in Kyma.
+
+## Event Publisher Proxy
+
+The Event Publisher Proxy component receives legacy and Cloud Event publishing requests from the cluster workloads (microservice or Serverless Functions) and redirects them to the Enterprise Messaging Service Cloud Event Gateway. It also fetches a list of subscriptions for a connected application.
+
+## Eventing Controller
+
+The Eventing Controller component manages the internal infrastructure in order to receive an event. The Controller watches subscription Custom Resource Definitions. When an event is received in an Application, it lays down the Eventing infrastructure in NATS in order to trigger a Function.
```

Comment:
Please mention that the NATS controller also dispatches Events to subscribers (microservices or Serverless Functions).